File: app/src/main/java/com/julio/api/model/DataItem.java
package com.julio.api.model;

import com.google.gson.annotations.SerializedName;

public class DataItem {

    @SerializedName("email")
    public String email;

    @SerializedName("id")
    public int id;

    @SerializedName("first_name")
    public String firstName;

    @SerializedName("last_name")
    public String lastName;

    @SerializedName("avatar")
    public String avatar;
}

File: app/src/main/java/com/julio/api/model/DataItem.kt
package com.julio.api.model

import com.google.gson.annotations.SerializedName

data class DataItem(

        @SerializedName("id")
        var id: Int = 0,

        @SerializedName("email")
        var email: String? = null,

        @SerializedName("first_name")
        var firstName: String? = null,

        @SerializedName("last_name")
        var lastName: String? = null,

        @SerializedName("avatar")
        var avatar: String? = null
)